task.sql

```diff
@@ -10,253 +10,124 @@ CREATE TABLE Products1 (
-INSERT INTO Products2 (Name)
-	VALUES ('AwersomeProduct14'); 
-INSERT INTO Products2 (Name)
-	VALUES ('AwersomeProduct15'); 
-INSERT INTO Products2 (Name)
-	VALUES ('AwersomeProduct16'); 
-INSERT INTO Products2 (Name)
-	VALUES ('AwersomeProduct17'); 
-INSERT INTO Products2 (Name)
-	VALUES ('AwersomeProduct18'); 
-INSERT INTO Products2 (Name)
-	VALUES ('AwersomeProduct19'); 
-INSERT INTO Products2 (Name)
-	VALUES ('AwersomeProduct20'); 
-INSERT INTO Products2 (Name)
-	VALUES ('AwersomeProduct21'); 
-INSERT INTO Products2 (Name)
-	VALUES ('AwersomeProduct22'); 
-INSERT INTO Products2 (Name)
-	VALUES ('AwersomeProduct23'); 
-INSERT INTO Products2 (Name)
-	VALUES ('AwersomeProduct24'); 
-INSERT INTO Products2 (Name)
-	VALUES ('AwersomeProduct25'); 
-INSERT INTO Products2 (Name)
-	VALUES ('AwersomeProduct26'); 
-INSERT INTO Products2 (Name)
-	VALUES ('AwersomeProduct27'); 
-INSERT INTO Products2 (Name)
-	VALUES ('AwersomeProduct28'); 
-INSERT INTO Products2 (Name)
-	VALUES ('AwersomeProduct29'); 
-INSERT INTO Products2 (Name)
-	VALUES ('AwersomeProduct30'); 
-INSERT INTO Products2 (Name)
-	VALUES ('AwersomeProduct31'); 
-INSERT INTO Products2 (Name)
-	VALUES ('AwersomeProduct32'); 
-INSERT INTO Products2 (Name)
-	VALUES ('AwersomeProduct33'); 
-INSERT INTO Products2 (Name)
-	VALUES ('AwersomeProduct34'); 
-INSERT INTO Products2 (Name)
-	VALUES ('AwersomeProduct35'); 
-INSERT INTO Products2 (Name)
-	VALUES ('AwersomeProduct36'); 
-INSERT INTO Products2 (Name)
-	VALUES ('AwersomeProduct37'); 
-INSERT INTO Products2 (Name)
-	VALUES ('AwersomeProduct38'); 
-INSERT INTO Products2 (Name)
-	VALUES ('AwersomeProduct39'); 
-INSERT INTO Products2 (Name)
-	VALUES ('AwersomeProduct40'); 
-INSERT INTO Products2 (Name)
-	VALUES ('AwersomeProduct41'); 
-INSERT INTO Products2 (Name)
-	VALUES ('AwersomeProduct42'); 
-INSERT INTO Products2 (Name)
-	VALUES ('AwersomeProduct43'); 
-INSERT INTO Products2 (Name)
-	VALUES ('AwersomeProduct44'); 
-INSERT INTO Products2 (Name)
-	VALUES ('AwersomeProduct45'); 
-INSERT INTO Products2 (Name)
-	VALUES ('AwersomeProduct46'); 
-INSERT INTO Products2 (Name)
-	VALUES ('AwersomeProduct47'); 
-INSERT INTO Products2 (Name)
-	VALUES ('AwersomeProduct48'); 
-INSERT INTO Products2 (Name)
-	VALUES ('AwersomeProduct49'); 
-INSERT INTO Products2 (Name)
-	VALUES ('AwersomeProduct50'); 
-INSERT INTO Products2 (Name)
-	VALUES ('AwersomeProduct51'); 
-INSERT INTO Products2 (Name)
-	VALUES ('AwersomeProduct52'); 
-INSERT INTO Products2 (Name)
-	VALUES ('AwersomeProduct53'); 
-INSERT INTO Products2 (Name)
-	VALUES ('AwersomeProduct54'); 
-INSERT INTO Products2 (Name)
-	VALUES ('AwersomeProduct55'); 
-INSERT INTO Products2 (Name)
-	VALUES ('AwersomeProduct56'); 
-INSERT INTO Products2 (Name)
-	VALUES ('AwersomeProduct57'); 
-INSERT INTO Products2 (Name)
-	VALUES ('AwersomeProduct58'); 
-INSERT INTO Products2 (Name)
-	VALUES ('AwersomeProduct59'); 
-INSERT INTO Products2 (Name)
-	VALUES ('AwersomeProduct60'); 
+	VALUES ('AwersomeProduct60');
```

Comment:
The task requires the creation of a second table, `Products2`, using a different database engine. Please add this table and populate it with data to meet the task requirements.